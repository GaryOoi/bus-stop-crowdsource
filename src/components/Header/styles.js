import styled from "styled-components";
import { Link } from "react-router-dom";

export const NavHeader = styled.h1`
  color: #fff;
  line-height: 1.5;
  font-size: 24px;
  font-weight: 800;
`;

export const StyledLink = styled(Link)`
  &:hover {
    text-decoration: none;
  }
`;
