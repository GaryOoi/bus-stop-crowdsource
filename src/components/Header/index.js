import React from "react";
import { Navbar, Container } from "react-bootstrap";

import { NavHeader, StyledLink } from "./styles";

function Header() {
  return (
    <Navbar bg="dark" variant="dark">
      <Container className="d-flex p-2">
        <StyledLink to="/">
          <NavHeader href="#home">Bus Stop Crowdsource</NavHeader>
        </StyledLink>
      </Container>
    </Navbar>
  );
}

export default Header;
