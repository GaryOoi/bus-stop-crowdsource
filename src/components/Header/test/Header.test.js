import React from "react";
import { MemoryRouter } from "react-router-dom";
import { render } from "@testing-library/react";

import Header from "../index";

test("renders learn react link", () => {
  const { container } = render(
    <MemoryRouter>
      <Header />
    </MemoryRouter>
  );

  expect(container.firstChild).toMatchSnapshot();
});
