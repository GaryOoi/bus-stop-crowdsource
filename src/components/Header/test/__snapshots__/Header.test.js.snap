// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<nav
  class="navbar navbar-expand navbar-dark bg-dark"
>
  <div
    class="d-flex p-2 container"
  >
    <a
      class="sc-AxirZ eEZWlz"
      href="/"
    >
      <h1
        class="sc-AxjAm iLGRWW"
        href="#home"
      >
        Bus Stop Crowdsource
      </h1>
    </a>
  </div>
</nav>
`;
